SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch

```diff
@@ -207,8 +207,8 @@ index 0000000..d797490
 +            try:
 +                self.mount()
 +                os.symlink(self.linkpath, self.path)
-+            except CephFSException, exc:
-+                raise SR.SROSError(12, exc.errstr)
++            except CephFSException as exc:
```

Comment:
It feels a bit strange to make this change in the first patch, when there are other patches that will be applied later to alter it for python3 support, IPv6 support, etc.

To be discussed when @Wescoeur is back?

Reply:
Yeah I think we need to re-do our patches to have less.